viewmaster/pdsgroup.py

```
@@ -18,23 +20,33 @@ class PdsGroup(object):
+
+    Attributes:
+        parent_pdsf: Parent `Pds3File` (False until initialized; None for
+            merged directories).
+        anchor (str|None): Group anchor derived from the first member unless
+            explicitly set.
+        rows (list): Ordered list of member PdsFiles.
+        hidden (set[str]): Logical paths of members hidden from default views.
     """
 
-    def __init__(self, pdsfiles=[], parent=False, anchor=None, hidden=[]):
+    def __init__(self, pdsfiles=None, parent=False, anchor=None, hidden=None):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)